### PR TITLE
feat(pagination): Adds MVP pagination to orders and products

### PR DESCRIPTION
**Pull request by olivermrbl**
Tests obviously do not pass at the moment. Will fix these, when we've solved the cookie issue.

### CODE REVIEW

File: packages/medusa-interfaces/src/base-model.js [hunk 1]
```diff
@@ -67,8 +67,16 @@ class BaseModel {
    * @return {Array<mongoose.Document>} the retreived mongoose documents or
    * an empty array
    */
-  find(query, options) {
-    return this.mongooseModel_.find(query, options).lean()
+  find(query, options, offset, limit) {
+    return this.mongooseModel_
+      .find(query, options)
+      .skip(offset)
+      .limit(limit)
```
Comment (srindom) [inline packages/medusa-interfaces/src/base-model.js:74]: Do these work when offset and limit are undefined?

File: packages/medusa/package.json [hunk 2]
```diff
@@ -50,6 +50,7 @@
     "body-parser": "^1.19.0",
     "bull": "^3.12.1",
     "chokidar": "^3.4.2",
+    "client-sessions": "^0.8.0",
```
Comment (srindom) [inline packages/medusa/package.json:53]: We don't use client-sessions anymore

**Review by srindom — APPROVED** [review]
🏇

File: packages/medusa-interfaces/src/base-model.js (hunk 1 above)
Comment (olivermrbl) [reply packages/medusa-interfaces/src/base-model.js:74]: Tested with `list-customers`, that does not have these yet. Still works :)

**Pull request merged**